lib/ini.js

```diff
@@ -1,25 +1,19 @@
 const { hasOwnProperty } = Object.prototype
 
-/* istanbul ignore next */
-const eol = typeof process !== 'undefined' &&
-  process.platform === 'win32' ? '\r\n' : '\n'
-
-const encode = (obj, opt) => {
-  const children = []
-  let out = ''
-
+const encode = (obj, opt = {}) => {
   if (typeof opt === 'string') {
-    opt = {
-      section: opt,
-      whitespace: false,
-    }
-  } else {
-    opt = opt || Object.create(null)
-    opt.whitespace = opt.whitespace === true
-    opt.newline = opt.newline === true
+    opt = { section: opt }
   }
+  opt.whitespace = opt.whitespace === true
+  opt.newline = opt.newline === true
+  /* istanbul ignore next */
+  opt.platform = opt.platform || process?.platform
```

Comment:
Trying to access `process` in a browser environment fails.
The availability of the value needs to be checked with `typeof process === 'undefined'`